source/content/email.md

```diff
@@ -21,7 +22,7 @@ Here are some popular email services you can use on the platform and their corre
 | Mailgun   | [Drupal](https://www.drupal.org/project/mailgun) \| [WordPress](https://wordpress.org/plugins/mailgun/) |
 | Mandrill  | [Drupal](https://www.drupal.org/project/mandrill) \| [WordPress](https://wordpress.org/plugins/wpmandrill/) |
 | Postmark | [Drupal](https://www.drupal.org/project/postmark) \| [WordPress](https://wordpress.org/plugins/postmark-approved-wordpress-plugin/) |
-| Sendgrid  | [Drupal](https://www.drupal.org/project/sendgrid_integration) \| [WordPress](/guides/sendgrid) |
+| Sendgrid  | [Drupal](https://www.drupal.org/project/sendgrid_integration) \| [WordPress](/guides/sendgrid-wordpress-wp-mail-smtp) |
```

Comment:
We are linking off to our docs here instead of WordPress docs?

Reply:
Yes, I thought I'd do that instead of linking to the new plugin, but happy to discuss/update :)